advisories/github-reviewed/2023/05/GHSA-r3fg-3r88-6x3f/GHSA-r3fg-3r88-6x3f.json

```diff
@@ -1,12 +1,12 @@
 {
   "schema_version": "1.4.0",
   "id": "GHSA-r3fg-3r88-6x3f",
-  "modified": "2023-05-10T21:25:28Z",
+  "modified": "2023-05-10T21:25:29Z",
   "published": "2023-05-10T21:25:28Z",
   "aliases": [
     "CVE-2023-2616"
```

Comment:
My view: This alias should be removed. The suggested changes in this file should not be merged.